packages/react-instantsearch-hooks-dom/src/widgets/Breadcrumb.tsx

```diff
@@ -6,11 +6,9 @@ import { Breadcrumb as BreadcrumbUiComponent } from '../ui/Breadcrumb';
 import type { BreadcrumbProps as BreadcrumbUiProps } from '../ui/Breadcrumb';
 import type { UseBreadcrumbProps } from 'react-instantsearch-hooks';
 
-export type BreadcrumbProps = Omit<
-  BreadcrumbUiProps,
-  'items' | 'hasItems' | 'createURL' | 'onNavigate' | 'translations'
```

Comment:
the omit is longer, but seems more correct. I wonder if we somehow can have a more static object of the "props we will pass to the component" rather than listing manually though, because it's indeed hard to make sure this stays correct. Maybe we can omit the required props (not sure if that's a good heuristic)

Reply:
I separated the internal and exposed props on components, to make the latter easy to use in widgets.

Not set on the naming of the internal props though (`Breadcrumbs_Internal_Props`), so feel free to jump in if you have ideas. I am also not fond of exporting the type but I need them for the ui tests.